pkg/kev/config/defaults.go

```diff
@@ -91,9 +91,12 @@ const (
 	// DefaultProbeInitialDelay default 1m (1 minute)
 	DefaultProbeInitialDelay = "1m"
 
-	// DefaultProbeRetries default 3. Number of retries for liveness probe command
+	// DefaultProbeRetries default 3. Number of retries for probe command
 	DefaultProbeRetries = 3
 
+	// DefaultSuccessThreshold default 1. Minimum consecutive successes for the probe to be considered successful
+	DefaultSuccessThreshold = 1
```

Comment:
small rename suggestion: `DefaultSuccessThreshold` -> `DefaultProbeSuccessThreshold` to keep it in line with the other probe defaults.

Reply:
Actually on a second thought perhaps we shouldn't place a default here as there is no equivalent for this in Compose spec. We can simply assume it's 1 by default. 

Instead we should consider adding a dedicated config key for probe success threshold in an extension driven config and have it override the default in applicable probe types.